perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1473,6 +1473,7 @@ sub user_edit : Path('user_edit') : Args(1) {
 
     $c->forward('fetch_all_bodies');
     $c->forward('fetch_body_areas', [ $user->from_body ]) if $user->from_body;
+    $c->cobrand->call_hook('admin_user_edit_extra_data');
```

Comment:
Does this also need to be in the user add controller, so it's on the new user form?